test/e2e/page-objects/flows/send-transaction.flow.ts

```diff
@@ -10,15 +10,15 @@ import SnapSimpleKeyringPage from '../pages/snap-simple-keyring-page';
  * @param driver - The webdriver instance.
  * @param recipientAddress - The recipient address.
  * @param amount - The amount of the asset to be sent in the transaction.
- * @param gasfee - The expected transaction gas fee.
- * @param totalfee - The expected total transaction fee.
+ * @param gasFee - The expected transaction gas fee.
+ * @param totalFee - The expected total transaction fee.
  */
-export const sendTransaction = async (
+export const sendTransactionToAddress = async (
   driver: Driver,
   recipientAddress: string,
   amount: string,
-  gasfee: string,
-  totalfee: string,
+  gasFee: string,
+  totalFee: string,
```

Comment:
I think maybe here we could do the same operation of using an object instead of single params :thinking: 
Given that this it is only used in 2 specs, the change won't be too big yet. If we want to do this later in time, where more tests use this, it would be more challenging. So maybe it's worth considering. 
What do you think?